### ISSUE TITLE
[RPP] Update Invoice Email Link and Invoice Page

### ISSUE THREAD

**Issue by ChrispinP** [cityofaustin/atd-data-tech#5768]
Possible bug when navigating to the invoice page for a completed RPP application. Need to verify link is working correctly and that invoice page details and formatting still looks good.

**Comment by moorerst** [comment]
@ChrispinP was this bug reported by someone?

**Comment by moorerst** [comment]
@ChrispinP 

1. entered test application 2021-3032 
2. received the email for Submitted-Pending Review clicked the Invoice link and it displayed correctly
3. changed the application status to Pending Payment received the email for Review and Approved - Pending Payment clicked the Invoice link and it displayed correctly
4. changed the application status to Complete - Permits Issued did not receive another email notification  

![image](https://user-images.githubusercontent.com/56606465/114448938-cc034500-9b99-11eb-98f4-c66ac3a941e1.png)

**Comment by ChrispinP** [comment]
@moorerst not sure yet if it is an actual bug, something Diana noticed from testing in the new builder so just want to make sure its still functioning after the switchover and mobile optimizations. So this is something we should QA during that time.

**Comment by ChrispinP** [comment]
This page is looking better and is working from the email link. Should do some more page/print formatting to get it to a better spot and print on a single page.

**Comment by moorerst** [comment]
@ChrispinP some suggestions for modifications on the Invoice Page:

- [ ] Move email address up to the Applicant Email title
- [ ] Move Applicant Type answer up to the Applicant Type title
- [ ] Move actual resident address under Resident Address title

![image](https://user-images.githubusercontent.com/56606465/116278364-0baa5f00-a74c-11eb-9fa1-5c4f623753c5.png)

**Comment by ChrispinP** [comment]
Looks like Combined Hangtags and Decals value was not populating, corrected conditional rule on field and updating records. Moved customer info to right side of page so on mobile its meets up with address now.

Mobile optimizations are affecting the print preview making everything bigger there as well. 
![image](https://user-images.githubusercontent.com/67915167/116313338-42e03680-a773-11eb-8ad8-6eef206efb5c.png)

I was able to clean up formatting some to get it to print on 2 pages. If we are happy with it being big we can leave as is or we would have to add code to control the print output separate from the mobile code.
